Feature 5

Namrata-J opened the pull request:
#5 added card component
*added basic card component
*added card with badges component
*added card with dismiss component
*added card with text overlay component
*added text only card component

Also, i have placed all these different card components in different files, so as to make it easy for the reviewer to review it!!
Please review it, thankyou!!

Namrata-J committed:
update: correction done in the-intro-line

Namrata-J committed:
added css files for card components into the common css file for all components

Namrata-J committed:
feat: added basic card component

Namrata-J committed:
style: added styling for basic card component

Namrata-J committed:
feat: added card with badges component

Namrata-J committed:
style: added style for card with badges component

Namrata-J committed:
feat: added card with dismiss component

Namrata-J committed:
style: added style for card with dismiss component

Namrata-J committed:
added js for card with dismiss component

Namrata-J committed:
feat: added card with text overlay component

Namrata-J committed:
style: added style for card with text overlay component

Namrata-J committed:
feat: added text only card component

Namrata-J committed:
style: added styling for text only card component

ozairkhan2810 commented on components/cards/basic-cards/basic-cards.css:
```diff
@@ -0,0 +1,206 @@
+    background-color: rgb(235, 245, 245);
+    border-radius: 0.7rem 0.7rem 0.3rem 0.2rem;
+    box-shadow:  15px 15px 20px 3px rgb(27, 27, 27);
+    height: 300px;
```
use rem to maintain consistency in units

mariyasada reviewed (approved):
good to merge

ozairkhan2810 commented on components/cards/basic-cards/basic-cards.css:
```diff
@@ -0,0 +1,206 @@
+
+/* basic w/ horizontal image and text component6 starts */
+.et_basic-whiat-card-container{
+    height: 120px;
```
use same units for consistency

Namrata-J replied:
okay will look into it!! thankyou so much for reviewing!!

ozairkhan2810 commented:
just a small suggestion to maintain consistency in unit, rest is good

ShreyashMahajan commented on components/cards/basic-cards/basic-cards.css:
```diff
@@ -0,0 +1,206 @@
+/* basic component styling*/
+.et_b-card-container {
+    background-color: rgb(235, 245, 245);
+    border-radius: 0.7rem 0.7rem 0.3rem 0.2rem;
```
try to use px for less than 1rem

ShreyashMahajan commented on components/cards/basic-cards/basic-cards.css:
```diff
@@ -0,0 +1,206 @@
+    height: 50%;
+}
+
+.et_b-card-subcontainer2 {
```
you can give naming for sub-container instead of number ( just a suggestion, please ignore this comment if not required)

ShreyashMahajan commented on components/cards/basic-cards/basic-cards.css:
```diff
@@ -0,0 +1,206 @@
+    opacity: 0.7;
+}
+
+button:hover {
```
button class can be used instead of the tag. otherwise, It will change the button CSS of the whole page.

ShreyashMahajan commented on components/cards/basic-cards/basic-cards.html:
```diff
@@ -0,0 +1,186 @@
+            </div>
+            <div class="et_b-card-subcontainer4 et_basic-woab-subcontainer4">
+                <div class="et_b-subcontainer4-action-buttons et_basic-woab-subcontainer4-action-buttons">
+                    <button>READ</button>
```
here you can use class for button to give css

Namrata-J replied:
okay will try to resolve it!!, thankyou so much for reviewing!!

ShreyashMahajan reviewed (approved):
code is very understandable.

Namrata-J commented:
> good to merge

Namrata-J committed:
fix: did the suggested changes

fixed the units. replaced the units defined in px to rem for maintaining the consistency.
